### PR TITLE
Rename eldeveloper-macfusion to macfusion-ng

### PR DESCRIPTION
**Pull request by gtmgianni**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by commitay** [comment]
Not sure what to call this now. `macfusion-ng`, ~~`macfusion-ng-macfusion`~~, `macfusion-ng-unoffical`

**Comment by vitorgalvao** [comment]
I’d say `macfusion-ng`.

File: Casks/macfusion-ng.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   sha256 '3e6e356f36623dde805aa3de941e29f6256b02cfe0720bcbd70df4526e2a7198'
 
-  url "https://github.com/ElDeveloper/macfusion#{version.major}/releases/download/#{version}/Macfusion.zip"
-  appcast "https://github.com/ElDeveloper/macfusion#{version.major}/releases.atom",
-          checkpoint: '4f19fdf3855e04e508dec355554487deaba49414556e6f93bc947d054703808b'
+  url "https://github.com/macfusion-ng/macfusion#{version.major}/releases/download/#{version}/Macfusion.zip"
+  appcast 'https://github.com/macfusion-ng/macfusion#{version.major}/releases.atom',
+          checkpoint: '501d54473744c385e7a060ff5890907ada340ce0f95ca48c2dd45c58947d3b53'
   name 'Macfusion'
   homepage "https://github.com/ElDeveloper/macfusion#{version.major}/"
```
Comment (commitay) [inline Casks/macfusion-ng.rb]: Please update the homepage

**Review by commitay — APPROVED** [review]
Verified locally.

**Pull request merged**